src/test/java/com/egov/core-services/pretests/pgServiceUpdate.feature

```diff
@@ -0,0 +1,58 @@
+Feature: Update a payment transaction
+
+Background:
+  * def jsUtils = read('classpath:jsUtils.js')
+  * def pgServicesUpdatePayload = read('../requestPayload/pgServices/pgServicesUpdate.json')
+
```

Comment:
create, update and search pretests should be in single file or it should be grouped under service name folder